boards/STM32F407ZET6-STM32_F4XX.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "STM32F407ZET6 - STM32_F4XX board"
+---
+
+# {{ page.title }}
+
+![STM32_F4XX board]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_1.jpg)
+
+## Overview
+
+<table>
+    <tr>
+        <td rowspan="3"><b>Board</b></td>
+        <td>Name</td>
+        <td>STM32_F4XX</td>
+    </tr>
+    <tr>
+        <td>Brand</td>
+        <td>Unknown</td>
+    </tr>
+    <tr>
+        <td>Origin</td>
+        <td>China</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Microcontroller</b></td>
+        <td>Part</td>
+        <td>STM32F407ZET6</td>
+    </tr>
+    <tr>
+        <td>Package</td>
+        <td>LQFP144</td>
+    </tr>
+    <tr>
+        <td>Core</td>
+        <td>Cortex-M4</td>
+    </tr>
+    <tr>
+        <td>FLASH</td>
+        <td>512KB (There is a STM32F4ZGT6 variant with 1024KB)</td>
+    </tr>
+    <tr>
+        <td>SRAM</td>
+        <td>128KB<br>64KB (CCM)<br>4KB (Backup)</td>
```

Comment:
Yay, I'll have to add this to all other board pages that have got devices with these or other types of memory

Reply:
Sorry about that, but I do think it's relevant.

Reply:
Oh, don't get me wrong, this was more a note-to-self. I'll have to check all board pages when I convert them to the new JSON based variant.